src/variorum/config_architecture.c

```diff
@@ -94,9 +98,12 @@ int variorum_detect_arch(void)
 #ifdef VARIORUM_WITH_IBM
     g_platform.ibm_arch = detect_ibm_arch();
 #endif
-#ifdef VARIORUM_WITH_GPU
+#ifdef VARIORUM_WITH_NVIDIA
```

Comment:
I think this ifdef block can be deleted (it has `detect_gpu_arch()` commented out in any case).

Reply:
Good point -- made the change.